### PR TITLE
Added --force

### PR DESCRIPTION
**Pull request by codeOfRobin**

### CODE REVIEW

File: main.swift [hunk 1]
```diff
@@ -77,6 +77,7 @@ struct Arguments {
     var githubURL: String?
     var organizationName: String?
     var repositoryURL: URL?
+	var isForce: Bool = false
```
Comment (JohnSundell) [inline main.swift]: Incorrect indentation here 😇

Comment (JohnSundell) [inline main.swift]: How about `forceEnabled` instead?

File: main.swift [hunk 2]
```diff
@@ -97,6 +98,8 @@ struct Arguments {
                 if let urlString = arguments.element(after: index) {
                     repositoryURL = URL(string: urlString)
                 }
+			case "--force", "-f":
```
Comment (JohnSundell) [inline main.swift]: Incorrect indentation here too 🙂

File: main.swift [hunk 3]
```diff
@@ -263,7 +266,7 @@ if let organizationName = organizationName {
 
 print("---------------------------------------------------------------------")
 
-if !askForBooleanInfo(question: "Proceed? ✅") {
+if !arguments.isForce && !askForBooleanInfo(question: "Proceed? ✅") {
```
Comment (JohnSundell) [inline main.swift]: How about nesting these if statements instead, makes it clear that `forceEnabled` is a precondition for `askForBooleanInfo()`.

File: main.swift (hunk 1 above)
Comment (codeOfRobin) [inline main.swift]: Okay, sure!

Comment (codeOfRobin) [inline main.swift]: Wait, LGTM on my machine. What sorcery is this 😱 ?
http://imgur.com/a/4BtWo

**Pull request merged**